### PR TITLE
Add dependencies needed for EDD scripts in setup.cfg

### PR DESCRIPTION
**Pull request by loichuder**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -5,9 +5,9 @@ description = Strain analysis
 long_description = file: README.md
 long_description_content_type = text/markdown
 license = MIT
-url = https://github.com/mohamedfaresslim/easistrain
+url = https://github.com/EASI-STRESS/easistrain
 project_urls = 
-    Bug Tracker = https://github.com/mohamedfaresslim/easistrain/issues
+    Bug Tracker = https://github.com/EASI-STRESS/easistrain/issues
```
Comment (loichuder) [inline setup.cfg:10]: I changed the URL to the org URL

File: setup.cfg [hunk 2]
```diff
@@ -18,7 +18,11 @@ packages = find:
 python_requires = >=3.6
 install_requires = 
     ewokscore
+    h5py
+    numpy
     pyfai
+    scipy
+    silx
```
Comment (loichuder) [inline setup.cfg:25]: These are needed to run `fitEDD.py`

File: setup.cfg [hunk 3]
```diff
@@ -27,6 +31,7 @@ console_scripts =
 [options.extras_require]
 test = 
     ewokscore[test]
+    pytest
```
Comment (loichuder) [inline setup.cfg:34]: I will use `pytest` to write/run tests

**Review by mohamedfaresslim — APPROVED** [review]
OK

**Pull request merged**